test/UnitTest/Microsoft.AspNetCore.OData.Test/Extensions/Endpoint/ODataEndpointRouteValueTransformerTests.cs

```diff
@@ -104,6 +107,73 @@ public void TransformAsyncReturnsCorrectRouteValues()
             Assert.Same(actionDescriptor, httpContext.ODataFeature().ActionDescriptor);
         }
 
+        [Theory]
+        [InlineData(false, true)]
+        [InlineData(false, false)]
+        public async void TransformAsyncReturnsCorrectRouteValuesForNonOptionRequests(bool isOptionsRequest, bool includeMetadata)
+        {
+            var endpoint = await GenerateCorsEndPoint(isOptionsRequest, includeMetadata);
+            // Assert the metadata is empty when options requests are sent and the controller is not annotated
+
+            Assert.Null(endpoint);
+        }
+
+        [Theory]
+        [InlineData(true, true)]
```

Comment:
>true [](start = 20, length = 4)

don't really have to pass this in since it's true for all cases in this test.

Reply:
resolved